myapp/logic/trade_logic.py

```diff
@@ -0,0 +1,34 @@
+from myapp import stock_api
+from myapp import models
+
+def extract_info(params):
+    try:
+        number_of_stocks = int(params["number_of_stocks"])
```

Comment:
dont be so flexible. If you want an int, this should be passed as an int

Reply:
I did not know html could provide that :) newbie at html.
I fixed it. Now it only submits numbers